data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -347,6 +360,7 @@ then
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
 {{- if .SingleNode.BootstrapInPlace}}
         start --tear-down-early=false --asset-dir=/assets --required-pods=""  --assets-create-timeout=5m
+        touch etcd-bootstrap-removed.done
```

Comment:
This isn't right,
cluster-bootstrap isn't removing etcd, we take down etcd static pod later during bootstrap-in-place.sh.